Sources/PagerTabStripView.swift

```diff
@@ -71,12 +71,12 @@ private struct WrapperPagerTabStripView<Content> : View where Content: View {
     
     public var body: some View {
         GeometryReader { gproxy in
-            HStack(spacing: 0) {
+            LazyHStack(spacing: 0) {
```

Comment:
What is the idea of the LazyStack? To not instantiate all pages? Does it work correctly? I remember seeing that they would still all be created but maybe that's not the case anymore